jena-sdb/assembly.xml

```diff
@@ -41,11 +41,11 @@
 
   <files>
     <file>
-      <source>dist/LICENSE</source>
+      <source>LICENSE</source>
```

Comment:
They are probably different.  The `dist/` version cover everything in the assembly, including shipped dependencies. The ones at the top of the module just cover the code in the module.

Can we just pull `dist/` out of git and check?